src/js/vendingMachine/itemManager.test.ts

```diff
@@ -7,7 +7,8 @@ describe('상품 이름, 가격, 수량이 주어질 때', () => {
     const name = '콜라';
     const price = 1200;
     const quantity = 1;
-    const item = new Item(name, price, quantity);
+    const id = 0;
+    const item = new Item(name, price, quantity, id);
 
     itemManager.addItem({ name, price, quantity });
```

Comment:
```suggestion
    const item = new Item(name, price, quantity, id);

    itemManager.addItem({ name, price, quantity, id });
```
id가 undefined네요~

새로운 기능을 추가할 때 기존 테스트가 동작하는지 테스트해보시면 좋을거같아요.
변경할때 테스트가 실패한 상태로 기존의 테스트를 방치한다면 테스트를 하는 이유가 없어보이네요
테스트를 하는 이유에 대해 한번 더 생각해볼 수 있으면 좋겠습니다 🙂

Reply:
기능 추가를 테스트를 하면서 했어야 했는데, 구현하는 것에 급했던 것 같다고 생각했습니다.😓